### ISSUE TITLE
feat: Current line highlight

### ISSUE THREAD

**Issue by hgigas** [lewis6991/gitsigns.nvim#733]
Thank you a lot for the plugin.

Would it be possible to have `GitSignsAddCursorLineNR`, `GitSignsChangeCursorLineNR` and `GitSignsDeleteCursorLineNR` for current line? I don't care about names you give them.

**Comment by unphased** [comment]
I use nvim-cursorline to get word highlighting and this has cursorline functionality. May help you. The functionality seems out of scope for this plugin.

**Comment by unphased** [comment]
Oh oh oh are you saying you want a special line highlight for the cursor's line, for when you are in add/change/delete lines (as worked out by this plugin)? That would be interesting.

**Comment by hgigas** [comment]
Yes, that's the point. I'm sorry my first comment wasn't clear enough.
